src/commands/new.ts

```diff
@@ -642,6 +678,15 @@ export default {
       p2()
     }
 
+    if (warnAboutEOL) {
+      hr()
+      p2()
+      p2(yellow(`Generator templates could not be converted to Windows EOL.`))
+      p2(yellow(`You may want to update these manually with your code editor, more info at:`))
+      p2(`${link("https://docs.infinite.red/ignite/concept/generators#windows")}`)
```

Comment:
This page 404s :( https://docs.infinite.red/ignite/concept/generators

Reply:
We can use this for now https://github.com/infinitered/ignite/blob/master/docs/Generators.md#windows

Reply:
Yeah I was just planning for the future lol, but we can do where it currently lives. The link I provided banks on this getting through: https://github.com/infinitered/ignite/pull/2530

Reply:
fixed